### ISSUE TITLE
Error when load sensors

### ISSUE THREAD

**Issue by hy-storai** [louislam/uptime-kuma#2324]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### Description

<details><summary>Logs</summary>
<p>

```raw
Nov 16 02:17:47 rs-up-host node[768]:   sql: undefined,
Nov 16 02:17:47 rs-up-host node[768]:   bindings: undefined
Nov 16 02:17:47 rs-up-host node[768]: }
Nov 16 02:17:47 rs-up-host node[768]:     at process.<anonymous> (/opt/uptime/server/server.js:1728:13)
Nov 16 02:17:47 rs-up-host node[768]:     at process.emit (node:events:513:28)
Nov 16 02:17:47 rs-up-host node[768]:     at emit (node:internal/process/promises:140:20)
Nov 16 02:17:47 rs-up-host node[768]:     at processPromiseRejections (node:internal/process/promises:274:27)
Nov 16 02:17:47 rs-up-host node[768]:     at processTicksAndRejections (node:internal/process/task_queues:97:32)
Nov 16 02:17:47 rs-up-host node[768]: If you keep encountering errors, please report to https://github.com/louislam/uptime-kuma/issues
Nov 16 02:17:47 rs-up-host node[768]: Trace: KnexTimeoutError: Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?
Nov 16 02:17:47 rs-up-host node[768]:     at Client_SQLite3.acquireConnection (/opt/uptime/node_modules/knex/lib/client.js:305:26)
Nov 16 02:17:47 rs-up-host node[768]:     at async Runner.ensureConnection (/opt/uptime/node_modules/knex/lib/execution/runner.js:259:28)
Nov 16 02:17:47 rs-up-host node[768]:     at async Runner.run (/opt/uptime/node_modules/knex/lib/execution/runner.js:30:19)
Nov 16 02:17:47 rs-up-host node[768]:     at async RedBeanNode.findOne (/opt/uptime/node_modules/redbean-node/dist/redbean-node.js:515:19)
Nov 16 02:17:47 rs-up-host node[768]:     at async Function.sendCertInfo (/opt/uptime/server/model/monitor.js:804:23)
Nov 16 02:17:47 rs-up-host node[768]:     at async Function.sendStats (/opt/uptime/server/model/monitor.js:769:13) {
Nov 16 02:17:47 rs-up-host node[768]:   sql: undefined,
Nov 16 02:17:47 rs-up-host node[768]:   bindings: undefined
Nov 16 02:17:47 rs-up-host node[768]: }
Nov 16 02:17:47 rs-up-host node[768]:     at process.<anonymous> (/opt/uptime/server/server.js:1728:13)
Nov 16 02:17:47 rs-up-host node[768]:     at process.emit (node:events:513:28)
Nov 16 02:17:47 rs-up-host node[768]:     at emit (node:internal/process/promises:140:20)
Nov 16 02:17:47 rs-up-host node[768]:     at processPromiseRejections (node:internal/process/promises:274:27)
Nov 16 02:17:47 rs-up-host node[768]:     at processTicksAndRejections (node:internal/process/task_queues:97:32)
Nov 16 02:17:47 rs-up-host node[768]: If you keep encountering errors, please report to https://github.com/louislam/uptime-kuma/issues
Nov 16 02:17:47 rs-up-host node[768]: Trace: KnexTimeoutError: Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?
Nov 16 02:17:47 rs-up-host node[768]:     at Client_SQLite3.acquireConnection (/opt/uptime/node_modules/knex/lib/client.js:305:26)
Nov 16 02:17:47 rs-up-host node[768]:     at async Runner.ensureConnection (/opt/uptime/node_modules/knex/lib/execution/runner.js:259:28)
Nov 16 02:17:47 rs-up-host node[768]:     at async Runner.run (/opt/uptime/node_modules/knex/lib/execution/runner.js:30:19)
Nov 16 02:17:47 rs-up-host node[768]:     at async RedBeanNode.findOne (/opt/uptime/node_modules/redbean-node/dist/redbean-node.js:515:19)
Nov 16 02:17:47 rs-up-host node[768]:     at async Function.sendCertInfo (/opt/uptime/server/model/monitor.js:804:23)
Nov 16 02:17:47 rs-up-host node[768]:     at async Function.sendStats (/opt/uptime/server/model/monitor.js:769:13) {
Nov 16 02:17:47 rs-up-host node[768]:   sql: undefined,
Nov 16 02:17:47 rs-up-host node[768]:   bindings: undefined
Nov 16 02:17:47 rs-up-host node[768]: }
Nov 16 02:17:47 rs-up-host node[768]:     at process.<anonymous> (/opt/uptime/server/server.js:1728:13)
Nov 16 02:17:47 rs-up-host node[768]:     at process.emit (node:events:513:28)
Nov 16 02:17:47 rs-up-host node[768]:     at emit (node:internal/process/promises:140:20)
Nov 16 02:17:47 rs-up-host node[768]:     at processPromiseRejections (node:internal/process/promises:274:27)
Nov 16 02:17:47 rs-up-host node[768]:     at processTicksAndRejections (node:internal/process/task_queues:97:32)
Nov 16 02:17:47 rs-up-host node[768]: If you keep encountering errors, please report to https://github.com/louislam/uptime-kuma/issues
Nov 16 02:17:47 rs-up-host node[768]: Trace: KnexTimeoutError: Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?
Nov 16 02:17:47 rs-up-host node[768]:     at Client_SQLite3.acquireConnection (/opt/uptime/node_modules/knex/lib/client.js:305:26)
Nov 16 02:17:47 rs-up-host node[768]:     at async Runner.ensureConnection (/opt/uptime/node_modules/knex/lib/execution/runner.js:259:28)
Nov 16 02:17:47 rs-up-host node[768]:     at async Runner.run (/opt/uptime/node_modules/knex/lib/execution/runner.js:30:19)
Nov 16 02:17:47 rs-up-host node[768]:     at async RedBeanNode.findOne (/opt/uptime/node_modules/redbean-node/dist/redbean-node.js:515:19)
Nov 16 02:17:47 rs-up-host node[768]:     at async Function.sendCertInfo (/opt/uptime/server/model/monitor.js:804:23)
Nov 16 02:17:47 rs-up-host node[768]:     at async Function.sendStats (/opt/uptime/server/model/monitor.js:769:13) {
Nov 16 02:17:47 rs-up-host node[768]:   sql: undefined,
Nov 16 02:17:47 rs-up-host node[768]:   bindings: undefined
Nov 16 02:17:47 rs-up-host node[768]: }
Nov 16 02:17:47 rs-up-host node[768]:     at process.<anonymous> (/opt/uptime/server/server.js:1728:13)
Nov 16 02:17:47 rs-up-host node[768]:     at process.emit (node:events:513:28)
Nov 16 02:17:47 rs-up-host node[768]:     at emit (node:internal/process/promises:140:20)
Nov 16 02:17:47 rs-up-host node[768]:     at processPromiseRejections (node:internal/process/promises:274:27)
Nov 16 02:17:47 rs-up-host node[768]:     at processTicksAndRejections (node:internal/process/task_queues:97:32)
```

</p>
</details> 

### 👟 Reproduction steps

Login and then appear

### 👀 Expected behavior

No errors

### 😓 Actual Behavior

Not show sensors

### 🐻 Uptime-Kuma Version

1.18.5

### 💻 Operating System and Arch

Ubuntu 22

### 🌐 Browser

Google Chrome

### 🟩 NodeJS Version

16

### 📝 Relevant log output

<details><summary>Logs</summary>
<p>

```raw
Nov 16 02:17:47 rs-up-host node[768]:   sql: undefined,
Nov 16 02:17:47 rs-up-host node[768]:   bindings: undefined
Nov 16 02:17:47 rs-up-host node[768]: }
Nov 16 02:17:47 rs-up-host node[768]:     at process.<anonymous> (/opt/uptime/server/server.js:1728:13)
Nov 16 02:17:47 rs-up-host node[768]:     at process.emit (node:events:513:28)
Nov 16 02:17:47 rs-up-host node[768]:     at emit (node:internal/process/promises:140:20)
Nov 16 02:17:47 rs-up-host node[768]:     at processPromiseRejections (node:internal/process/promises:274:27)
Nov 16 02:17:47 rs-up-host node[768]:     at processTicksAndRejections (node:internal/process/task_queues:97:32)
Nov 16 02:17:47 rs-up-host node[768]: If you keep encountering errors, please report to https://github.com/louislam/uptime-kuma/issues
Nov 16 02:17:47 rs-up-host node[768]: Trace: KnexTimeoutError: Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?
Nov 16 02:17:47 rs-up-host node[768]:     at Client_SQLite3.acquireConnection (/opt/uptime/node_modules/knex/lib/client.js:305:26)
Nov 16 02:17:47 rs-up-host node[768]:     at async Runner.ensureConnection (/opt/uptime/node_modules/knex/lib/execution/runner.js:259:28)
Nov 16 02:17:47 rs-up-host node[768]:     at async Runner.run (/opt/uptime/node_modules/knex/lib/execution/runner.js:30:19)
Nov 16 02:17:47 rs-up-host node[768]:     at async RedBeanNode.findOne (/opt/uptime/node_modules/redbean-node/dist/redbean-node.js:515:19)
Nov 16 02:17:47 rs-up-host node[768]:     at async Function.sendCertInfo (/opt/uptime/server/model/monitor.js:804:23)
Nov 16 02:17:47 rs-up-host node[768]:     at async Function.sendStats (/opt/uptime/server/model/monitor.js:769:13) {
Nov 16 02:17:47 rs-up-host node[768]:   sql: undefined,
Nov 16 02:17:47 rs-up-host node[768]:   bindings: undefined
Nov 16 02:17:47 rs-up-host node[768]: }
Nov 16 02:17:47 rs-up-host node[768]:     at process.<anonymous> (/opt/uptime/server/server.js:1728:13)
Nov 16 02:17:47 rs-up-host node[768]:     at process.emit (node:events:513:28)
Nov 16 02:17:47 rs-up-host node[768]:     at emit (node:internal/process/promises:140:20)
Nov 16 02:17:47 rs-up-host node[768]:     at processPromiseRejections (node:internal/process/promises:274:27)
Nov 16 02:17:47 rs-up-host node[768]:     at processTicksAndRejections (node:internal/process/task_queues:97:32)
Nov 16 02:17:47 rs-up-host node[768]: If you keep encountering errors, please report to https://github.com/louislam/uptime-kuma/issues
Nov 16 02:17:47 rs-up-host node[768]: Trace: KnexTimeoutError: Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?
Nov 16 02:17:47 rs-up-host node[768]:     at Client_SQLite3.acquireConnection (/opt/uptime/node_modules/knex/lib/client.js:305:26)
Nov 16 02:17:47 rs-up-host node[768]:     at async Runner.ensureConnection (/opt/uptime/node_modules/knex/lib/execution/runner.js:259:28)
Nov 16 02:17:47 rs-up-host node[768]:     at async Runner.run (/opt/uptime/node_modules/knex/lib/execution/runner.js:30:19)
Nov 16 02:17:47 rs-up-host node[768]:     at async RedBeanNode.findOne (/opt/uptime/node_modules/redbean-node/dist/redbean-node.js:515:19)
Nov 16 02:17:47 rs-up-host node[768]:     at async Function.sendCertInfo (/opt/uptime/server/model/monitor.js:804:23)
Nov 16 02:17:47 rs-up-host node[768]:     at async Function.sendStats (/opt/uptime/server/model/monitor.js:769:13) {
Nov 16 02:17:47 rs-up-host node[768]:   sql: undefined,
Nov 16 02:17:47 rs-up-host node[768]:   bindings: undefined
Nov 16 02:17:47 rs-up-host node[768]: }
Nov 16 02:17:47 rs-up-host node[768]:     at process.<anonymous> (/opt/uptime/server/server.js:1728:13)
Nov 16 02:17:47 rs-up-host node[768]:     at process.emit (node:events:513:28)
Nov 16 02:17:47 rs-up-host node[768]:     at emit (node:internal/process/promises:140:20)
Nov 16 02:17:47 rs-up-host node[768]:     at processPromiseRejections (node:internal/process/promises:274:27)
Nov 16 02:17:47 rs-up-host node[768]:     at processTicksAndRejections (node:internal/process/task_queues:97:32)
Nov 16 02:17:47 rs-up-host node[768]: If you keep encountering errors, please report to https://github.com/louislam/uptime-kuma/issues
Nov 16 02:17:47 rs-up-host node[768]: Trace: KnexTimeoutError: Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?
Nov 16 02:17:47 rs-up-host node[768]:     at Client_SQLite3.acquireConnection (/opt/uptime/node_modules/knex/lib/client.js:305:26)
Nov 16 02:17:47 rs-up-host node[768]:     at async Runner.ensureConnection (/opt/uptime/node_modules/knex/lib/execution/runner.js:259:28)
Nov 16 02:17:47 rs-up-host node[768]:     at async Runner.run (/opt/uptime/node_modules/knex/lib/execution/runner.js:30:19)
Nov 16 02:17:47 rs-up-host node[768]:     at async RedBeanNode.findOne (/opt/uptime/node_modules/redbean-node/dist/redbean-node.js:515:19)
Nov 16 02:17:47 rs-up-host node[768]:     at async Function.sendCertInfo (/opt/uptime/server/model/monitor.js:804:23)
Nov 16 02:17:47 rs-up-host node[768]:     at async Function.sendStats (/opt/uptime/server/model/monitor.js:769:13) {
Nov 16 02:17:47 rs-up-host node[768]:   sql: undefined,
Nov 16 02:17:47 rs-up-host node[768]:   bindings: undefined
Nov 16 02:17:47 rs-up-host node[768]: }
Nov 16 02:17:47 rs-up-host node[768]:     at process.<anonymous> (/opt/uptime/server/server.js:1728:13)
Nov 16 02:17:47 rs-up-host node[768]:     at process.emit (node:events:513:28)
Nov 16 02:17:47 rs-up-host node[768]:     at emit (node:internal/process/promises:140:20)
Nov 16 02:17:47 rs-up-host node[768]:     at processPromiseRejections (node:internal/process/promises:274:27)
Nov 16 02:17:47 rs-up-host node[768]:     at processTicksAndRejections (node:internal/process/task_queues:97:32)
```

</p>
</details>

**Comment by Computroniks** [comment]
This might be a known issue. How many monitors do you have running? If there are a lot, the database is known to get overloaded when loading large amounts of data, hopefully this should be fixed in 2.0.0 when we start to support mariadb/mysql.

**Comment by CommanderStorm** [comment]
Closing as @hy-storai did not respond with the nessesesary information to further debug this
